sdk/mediaservices/arm-mediaservices/README.md

```diff
@@ -1,93 +1,101 @@
 ## Azure AzureMediaServices SDK for JavaScript
 
-This package contains an isomorphic SDK for AzureMediaServices.
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for AzureMediaServices.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
```

Comment:
We are releasing an update to the code generator today that will result in the below changes. Suggesting this here so that you don't have to re-generate

```suggestion
- [LTS versions of Node.js](https://nodejs.org/about/releases/)
- Latest versions of Safari, Chrome, Edge, and Firefox.
```